Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
   "examples/dyn_create_destroy_apps",
   "examples/file_upload",
   "examples/function_todomvc",
+  "examples/function_memory_game",
```

Comment:
Sorted alphabetically:

```suggestion
  "examples/function_memory_game",
  "examples/function_todomvc",
```

Hope I didn't mess up the indentation on my phone

Reply:
sure, will update